src/input_parsers/PropertyParser.cpp

```diff
@@ -138,7 +138,7 @@ void PropertyParser::processSingleLine( const String &line, InputQuery &inputQue
 
             auto subToken = subTokens.begin();
             ++subToken;
-            unsigned layerIndex = atoi( subToken->ascii() );
+            unsigned layerIndex = 2 * atoi( subToken->ascii() ) - 1;
```

Comment:
Hmm...
So, it seems like this change is tightly coupled to the ACAS Xu architecture, of input, <fc,relu>*, output layers. Our NLR now supports general feed-forward topologies. I understand the motivation was to maintain the old ws indexing, but I'm not sure it's good to hard-code this.